packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -54,7 +54,7 @@ export class GutenbergEditorPage {
 	 */
 	async waitUntilLoaded(): Promise< Frame > {
 		const frame = await this.getEditorFrame();
-		await this.page.waitForLoadState( 'networkidle' );
+		await this.page.waitForLoadState( 'networkidle', { timeout: 60000 } );
```

Comment:
I noticed the iframe'd editor may take a long time to appear. The extended timeout is to help guard against an otherwise normally loading page.